docs/html/_sources/quick_start.rst.txt

```diff
@@ -48,19 +48,19 @@ This provides us with lots of information about percentage count of missing valu
 In this case, we might have been able to figure this out quickly from the call to head,
 but in larger datasets this might be a bit tricky.
 For example we can see that there are several dirty columns with "NaN" in it.
-let's try and continue with what slik is doing
-automatically for now. Now we can handle missing values with slik.
+let's try and continue with what Slik-ML is doing
+automatically for now. Now we can handle missing values with Slik-ML.
 
-    >>> slik.preprocessing.handle_nan(dataframe=titanic,target_name='Survived',strategy='mean',fillna='mode',
+    >>> Slik-ML.preprocessing.handle_nan(dataframe=titanic,target_name='Survived',strategy='mean',fillna='mode',
```

Comment:
I think we should stick to the lower case to avoid trouble. slik-ml should be fine.

Reply:
I changed it to lower cases where necessary